### DISCUSSION TITLE
Error "/config/ana-cont_11.3.1_s2.tflite"

### DISCUSSION THREAD

**Discussion by Toki99** [jomjol/AI-on-the-edge-device#2060] [Q&A]
I crashed my device after a manual Update from an older Firmware (html und Firmaware copy)
After that I setup the device new. I ereased the device before. Now i get the same Error.
What can i do?

In the selected field the value '/config/ana-cont_11.3.1_s2.tflite' in the section 'Analog' in the field 'Model' is invalid. PLEASE CHECK BEFORE SAVING!

**Comment by jomjol** [comment]
Then this file is not existing. Either upload it manually or choose another ana-..., which is available. As long as the verion is higher as the cited one (11.3.1) everything should be fine.

**Reply by caco3** [reply]
> https://jomjol.github.io/AI-on-the-edge-device-docs/

You should not do that as the html files get modified during the build process. Just use the provided zip file!

@jomjol I also have seen that the device crashes when a not-existing model is selected. We should add a check for this in the firmware.

**Reply by Slider0007** [reply]
> > https://jomjol.github.io/AI-on-the-edge-device-docs/
> 
> You should not do that as the html files get modified during the build process. Just use the provided zip file!
> 
> @jomjol I also have seen that the device crashes when a not-existing model is selected. We should add a check for this in the firmware.

@jomjol, @caco3 : For your info. I've already started with adding error handling to first main flow parts, flow init as well as e.g. tflite handling. This work should also cover missing tflite files. I'm in the doing right now and I can provide a proposal in the next time.

BTW, there are still a lot more places were error handling would be beneficial...

**Reply by caco3** [reply]
nice!

> BTW, there are still a lot more places were error handling would be beneficial...

es, eg. on parameter validation (ranges, existing files/folders, ...)
